### ISSUE TITLE
2017-06-28

### ISSUE THREAD

**Issue by andkov** [IALSA/ialsa-2017-amsterdam#10]
At this moment 5 studies have submitted models with the following predictors:

```r
> unique(ls_odds$octo$predictor) %>% as.character()
[1] "age"    "eduyrs" "cogact" "numfr"  "male"  

> unique(ls_odds$lasa$predictor) %>% as.character()
[1] "age"       "male"      "education" "cogpartic" "network"  

> unique(ls_odds$wh$predictor)%>% as.character()
[1] "age"      "male"     "tedtotyr" "leisure"  "netw"    

> unique(ls_odds$h70$predictor)%>% as.character()
[1] "age"     "sex"     "EDUYR"   "cogcomp" "socsize"

> unique(ls_odds$map$predictor)%>% as.character()
[1] "age"        "male"       "edu"        "cogact_old" "soc_net"  
```

# Question
What are good labels for study-specific names of `cognitive_activity` and `social_network_size`? What should the names of these two harmonized variables be?

# Tables
VIew tables in a [Word](https://rawgit.com/IALSA/ialsa-2017-amsterdam/master/reports/model-summaries/table-2.docx)  file

**Comment by andkov** [comment]
# View (A)
![a-transition-by-predictor](https://user-images.githubusercontent.com/2608073/27644375-5decdbf8-5bf1-11e7-9012-47119b1be0fa.png)

# View (B)
![b-study-by-predictor](https://user-images.githubusercontent.com/2608073/27644385-6076f1ec-5bf1-11e7-933b-d0ff52b867b8.png)


# View (C)
![c-study-by-transition](https://user-images.githubusercontent.com/2608073/27644391-638a0c8e-5bf1-11e7-8ef7-0d977a4c745c.png)

**Comment by andkov** [comment]
Sample size used in the model object

```r
OCTO ---565
LASA ---2506
WH ---1285
H70 ---549
MAP ---1718
```

**Comment by andkov** [comment]
#Inegrated models from Johan

# VIew A
![a-transition-by-predictor](https://user-images.githubusercontent.com/2608073/27646337-59369824-5bf6-11e7-885b-63de56335e39.png)

# VIew B
![b-study-by-predictor](https://user-images.githubusercontent.com/2608073/27646338-593b6638-5bf6-11e7-88de-de300a2e655e.png)

# View C
![c-study-by-transition](https://user-images.githubusercontent.com/2608073/27646339-5944f32e-5bf6-11e7-8408-038dc81003ad.png)

**Comment by andkov** [comment]
whitehall has no `3-->2` transition, is this ok, @annierobi?
![image](https://user-images.githubusercontent.com/2608073/27842598-510d87e2-60d9-11e7-8535-1b60aa4bed3e.png)

**Comment by annierobi** [comment]
Yes it's ok for Whitehall because she was unable to run the models without
fixing some parameters.

**Comment by andkov** [comment]
@annierobi , there re-made graphs and tables are ready.

# WITHOUT Whitehall
- [table](https://rawgit.com/IALSA/ialsa-2017-amsterdam/424a56209f8fbdb0f5eca05a9006db72e6a475a4/reports/model-summaries/table-2.docx)
- [View A: transition by predictor](https://rawgit.com/IALSA/ialsa-2017-amsterdam/424a56209f8fbdb0f5eca05a9006db72e6a475a4/reports/model-summaries/graphs/A-transition-by-predictor.png)
- [View C: study by transition](https://rawgit.com/IALSA/ialsa-2017-amsterdam/424a56209f8fbdb0f5eca05a9006db72e6a475a4/reports/model-summaries/graphs/C-study-by-transition.png)

# WITH Whitehall
- [table](https://rawgit.com/IALSA/ialsa-2017-amsterdam/cf4ecbb41abf7d24172a6f273c1ab1f9b7acf342/reports/model-summaries/table-2.docx)
- [View A: transition by predictor](https://rawgit.com/IALSA/ialsa-2017-amsterdam/cf4ecbb41abf7d24172a6f273c1ab1f9b7acf342/reports/model-summaries/graphs/A-transition-by-predictor.png)
- [View C: study by transition](https://rawgit.com/IALSA/ialsa-2017-amsterdam/cf4ecbb41abf7d24172a6f273c1ab1f9b7acf342/reports/model-summaries/graphs/C-study-by-transition.png)
